feat: Mention community images in reference (#213)
citynetwork/docs: docs/reference/images/index.md

* `os_version=${VERSION_ID}`: operating system version, such as `os_version=22.04`

[Other properties](https://docs.openstack.org/glance/latest/admin/useful-image-properties.html) may also be set on individual images.

### Community images

> At {{brand}} we regulary update and rotate our images to always provide

Reviewer: ```suggestion
At {{brand}}, we regularly update and rotate our images to always provide
```

secure public images.

> During rotation old images are moved from `public` to `community` visability while
> keeping same images names. This enables tools like OpenStack Heat or HashiCorp
> Terraform to pass validation checks without attempting to alter environments.

Reviewer: ```suggestion
During rotation, we change an image's visibility from `public` to `community`, while
keeping its image name. This enables tools like [Heat](https://docs.openstack.org/heat/)
or [Terraform](https://www.terraform.io/) to pass validation checks without attempting to alter environments.
```

> You still can check image original build date for both `public` and `community` images
> by checking `build_date` tag or `image_build_date` property that will have date in
> `YYYY-mm-dd` format.

Reviewer: ```suggestion
You can retrieve an image's original build date (for both `public` and `community` images)
by checking its `build_date` tag or `image_build_date` property.
```
